python/setup.cfg

```diff
@@ -3,3 +3,6 @@ description-file = README.md
 
 [flake8]
 max-line-length = 160
+exclude = 
+    ./.git,
+    ./venv
```

Comment:
nit: venv as a subdirectory is very specific/subjective. This might surprise some.

Reply:
Understood, however, this is also referenced in the Makefile https://github.com/metaparticle-io/package/blob/master/python/Makefile